### PR TITLE
Simplify build

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,10 +1,12 @@
 version: 2.1
 
 orbs:
-
   hmpps: ministryofjustice/hmpps@10
-  jira: circleci/jira@1.2.2
-  slack: circleci/slack@4.4.2
```
Comment (markreesmoj) [inline .circleci/config.yml:7]: both these orbs are included by the hmpps orb

File: .circleci/config.yml [hunk 2]
```diff
@@ -49,9 +51,6 @@ jobs:
             - build/libs
             - build.gradle.kts
             - app_version.txt
-      - slack/notify:
-          event: fail
-          template: basic_fail_1
```
Comment (markreesmoj) [inline .circleci/config.yml:54]: Don't see the point in this notification - devs will notice if their build fails pretty quickly

File: .circleci/config.yml [hunk 3]
```diff
@@ -157,9 +168,15 @@ workflows:
                 - main
     jobs:
       - hmpps/gradle_owasp_dependency_check:
+          slack_channel: << pipeline.parameters.alerts-slack-channel >>
```
Comment (markreesmoj) [inline .circleci/config.yml:171]: alerting when these fail, so developers notice and can take remedial action

**Review by johnyfarrar — APPROVED** [review]
Great work buddy!

**Pull request merged**